frontend/libs/ui-modules/src/modules/products/categories/components/CategoriesInline.tsx

```
@@ -0,0 +1,149 @@
+        ...category,
+        attachment: category.attachment
+          ? {
+              url: category.attachment.url,
+              name: '',
+            }
+          : undefined,
+        avatar: undefined,
+      };
+      updateCategories?.([...newCategories, transformedCategory]);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)